### DISCUSSION TITLE
Fatal error: Call to undefined function mb_internal_encoding() under Ubuntu Linux

### DISCUSSION THREAD

**Discussion by ghost** [datenstrom/community#249] [Report a bug] [answered]
host:/opt/lampp/htdocs/test/yellow-master$ php yellow.php build /opt/lampp/htdocs/test/yellow-master/static
PHP Fatal error:  Uncaught Error: Call to undefined function mb_internal_encoding() in /opt/lampp/htdocs/test/yellow-master/system/plugins/core.php:3366
Stack trace:
#0 /opt/lampp/htdocs/test/yellow-master/yellow.php(8): require_once()
#1 {main}
  thrown in /opt/lampp/htdocs/test/yellow-master/system/plugins/core.php on line 3366
host:/opt/lampp/htdocs/test/yellow-master$

**Comment by markseuffert** [comment]
Thanks for the bug report.

Which operating system do you use?
Which PHP version do you use?

**Comment by ghost** [comment]
PHP 7.0.18
Ubuntu 17.04

**Comment by markseuffert** [comment, marked as answer]
It looks like the `mbstring` extension is missing or not enabled. Here's how to install it:

`sudo apt-get install php-mbstring php7.0-mbstring`

If it's still not working, you might need to enable it manually:

`sudo phpenmod mbstring`

Hope this helps. 😄

**Comment by ghost** [comment]
Yes install of mbstring resolved the problem. Tack så mycket!

**Comment by markseuffert** [comment]
Good to hear. I added a better error message to Yellow, to help people with the installation.

Let us know if you find more bugs.
